.github/workflows/cibuildwheel.yml

```diff
@@ -26,9 +36,26 @@ jobs:
         with:
           python-version: '3.7'
 
+      - uses: actions/cache@v2
+        id: numpy-aarch64-cache
+        with:
+          path: numpy-aarch64-cache/
+          key: numpy-aarch64-cache
```

Comment:
This key needs some versioning, probably from some global variable that encodes the NumPy version.